index.js

```diff
@@ -38,9 +39,10 @@ Settings.loadRepositoriesSettings('./config.json')
                 'User-Agent': 'cesium-concierge',
                 Authorization: 'token ' + repositorySettings.gitHubToken
             };
-            var checkChangesMd = defined(repositorySettings.checkChangesMd) ? repositorySettings.checkChangesMd : true;
+            var checkChangesMd = parseBoolean(repositorySettings.checkChangesMd);
```

Comment:
This should be handled in Settings, just like we do with thirdParty folders.  It should always be defined by this point.